src/tsolvers/liasolver/LIASolver.h

```diff
@@ -59,10 +59,11 @@ class LIASolver: public LASolver
     TRes check    ( bool complete) override; // Checks the satisfiability of current constraints //PS. add the implementation to LIASolver.C
     void getNewSplits(vec<PTRef>& splits) override;
 
+    PTRef getInterpolant( const ipartitions_t &, map<PTRef, icolor_t>*, PartitionManager & );
```

Comment:
You are right. This seems to be related to the different ways how partitioning can be computed.
The signature was copied from LRASolver::getInterpolant, where it allows you to choose the way.
However, I think for LIA, only one way works, I think.
Since we don't have an agreement on a universal interpolation interface, I think I can remove the redundant parameters here.

Reply:
OK good, I agree.